frontend/src/containers/content/ContentDetailPage.js

```diff
@@ -1,59 +1,140 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useLocation } from 'react-router-dom';
 import ReviewList from '../../components/review/ReviewList';
+import { addFavoriteContent, getContentDetail } from '../../store/ContentStore';
+import { createReview } from '../../store/ReviewStore';
+import './ContentDetailPage.scss';
+// import posterTmp from './temp/best_offer.png';
 
 const ContentDetailPage = ({ history }) => {
-  const [content, setContent] = useState('');
+  const location = useLocation();
+  const path = location.pathname.split('/')[2];
```

Comment:
@muhwagwa How about useParams?
```suggestion
  import { useParams } from 'react-router-dom';
  const { id } = useParams();
```